backend/protocol_rpc/message_handler/redis_worker_handler.py

```
@@ -70,6 +71,51 @@ async def initialize(self):
+            await redis_pubsub.subscribe(self.VALIDATOR_CHANNEL)
+            logger.info(
+                f"Worker {self.worker_id} subscribed to {self.VALIDATOR_CHANNEL}"
+            )
+
+            try:
+                async for message in redis_pubsub.listen():
+                    if message["type"] == "message":
+                        try:
+                            event_data = json.loads(message["data"])
+                            await callback(event_data)
+                        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)